src/port/Game.cpp

```diff
@@ -325,7 +313,7 @@ s32 CM_GetCrossingOnTriggered(uintptr_t* crossing) {
 
 void CM_LoadTextures() {
     if (gWorldInstance.CurrentCourse) {
-        gWorldInstance.CurrentCourse->LoadTextures();
+        // gWorldInstance.CurrentCourse->LoadTextures();
```

Comment:
Delete CM_LoadTextures if not use?

I can do it in a later PR if you don't want. it's fine.

Reply:
hmm forget that I have comment it, I think I will leave it to you so like that you can complement with an unload function